dhall/tests/format/letCommentsB.dhall

```diff
@@ -0,0 +1,7 @@
+let {- Example documentation for a let binding
+       that spans more than one line
+    -}
+    x {- we can interject things here -} = -- Make sure one-line comments work
+            1
+
+in  x
```

Comment:
Could you add tests for cases like these?

```
let x {- foo                                        -} : {- bar                                           -} Bool {- baz                           -} = {- bla                      -} True
```

```
let x = True
let {- 0 -} y {- 1 -} : {- 2 -} = {- 3-} False
in x
```

```
let x = 1
in
let {- aaaaaaaaaaaaaaaaaaa
    -} y : {- bbbbbbbbbbbbbbbbbbbbb
    -} 2 {- cccccccccccccc
    -} = {- ddddddddddddddddd -}
    in x

Formatting stuff like this in a nice way seems tricky, but it would be good to at least record what it looks like.